docs/docs/cmd/aad/user/user-set.md

```diff
@@ -74,9 +74,9 @@ m365 aad user set [options]
 
 ## Remarks
 
-This command allows using unknown options.
+This command allows using unknown options. For a comprehensive list of user properties, please refer to the [Graph documentation page](https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties).
```

Comment:
To ensure I make the correct adjustments, I'd like to confirm if you are suggesting a change in the user-add.md file by removing the third admonition and incorporating its information into the existing content, resulting in the updated file structure as follows:

`## Remarks

!!! important
    To use this command you must be a Global administrator, User administrator or Privileged Authentication administrator

!!! note
    After running this command, it may take a minute before the user is effectively created in the tenant.

This command allows using unknown options. For a comprehensive list of user properties, please refer to the [Graph documentation page](https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties).`

Reply:
Yes correct, and also:

> if the specified option is not found, you will receive a `Resource 'xyz' does not exist or one of its queried reference-property objects are not present.` error.

Which I mentioned in a later comment